instat/sdgPlots.vb

```diff
@@ -605,6 +606,27 @@ Public Class sdgPlots
         SetFacetParameters()
     End Sub
 
+    Private Function GetAxisType(bIsX As Boolean) As String
+        Dim strAes As String
+
+        If bIsX Then
+            strAes = "x"
+        Else
+            strAes = "y"
+        End If
+        If clsGlobalAesFunction IsNot Nothing Then
+            If clsGlobalAesFunction.ContainsParameter(strAes) Then
+                'Run R code to determine type
+                'Temp default to continuous
+                Return "Continuous"
+            Else
+                'When aes not present discrete scale function works
+                Return "Discrete"
+            End If
+        Else
+            Return "Discrete"
+        End If
+    End Function

```

Comment:
This function checks the column type of the x or y axis which is needed to tell the axes control whether the scales function should be continuous, discrete, date etc. 
When there's no value discrete works. Currently this is returning continuous by default when there is a value in the global aes but eventually this should be checking the type by calling an R method. This can be done for further releases.